x-pack/legacy/plugins/ml/public/components/anomalies_table/severity_cell/severity_cell.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, memo } from 'react';
+import { EuiHealth } from '@elastic/eui';
+import { getSeverityColor } from '../../../../common/util/anomaly_utils';
+
+interface SeverityCellProps {
+  /**
+   * Severity score.
+   */
+  score: number;
+  /**
+   * Multi-bucket impact score from –5 to 5.
+   * Anomalies with a multi_bucket_impact value of greater than or equal
+   * to 2 are considered multi-bucket impact anomalies.
+   */
+  multiBucketImpact: number;
+}
+
+/**
+ * Renders anomaly severity score with single or multi-bucket impact marker.
+ */
+export const SeverityCell: FC<SeverityCellProps> = memo(({ score, multiBucketImpact }) => {
+  const severity = score >= 1 ? Math.floor(score) : '< 1';
+  const color = getSeverityColor(score);
+  const isMultiBucket = multiBucketImpact >= 2;
+  return isMultiBucket ? (
+    <div className="euiFlexGroup euiFlexGroup--gutterExtraSmall euiFlexGroup--alignItemsCenter euiFlexGroup--directionRow">
```

Comment:
I suspect this could make use of the EUI component equivalents (`EuiFlexGroup`, `EuiFlexItem`) and corresponding attributes